File: eTuristickaAgencija.Models/Korisnik.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTuristickaAgencija.Models
{
    public class Korisnik
    {
        public int Id { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string KorisnikoIme { get; set; }
        public byte[] Slika { get; set; }
        public string Email { get; set; }
        public int? UlogaId { get; set; }
        public Uloga Uloga { get; set; }

    }
}

File: UI/eturistickaagencija_admin/lib/models/korisnik.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnik.g.dart';

@JsonSerializable()
class Korisnik{
int? id;
String? ime;
String? prezime;
String? korisnikoIme;
String? slika;
String? email;
String? password;
String? passwordPotvrda;
int? ulogaId;


Korisnik(this.id,this.ime,this.prezime,this.korisnikoIme,this.slika,this.email,this.password,this.passwordPotvrda,this.ulogaId);

factory Korisnik.fromJson(Map<String,dynamic>json)=>_$KorisnikFromJson(json);
Map<String,dynamic>toJson()=>_$KorisnikToJson(this);
}
